commit ff005c77d76c18c514be6c88f52cb91b530a4868
Author: Jianfeng Wang <pkueewjf@gmail.com>
Date:   Thu Nov 2 17:56:57 2023 -0700

    nsl cluster prep

diff --git a/caladan/iokernel/hw_timestamp.c b/caladan/iokernel/hw_timestamp.c
index 71c1b28..b0f3702 100644
--- a/caladan/iokernel/hw_timestamp.c
+++ b/caladan/iokernel/hw_timestamp.c
@@ -87,7 +87,7 @@ int hw_timestamp_init(void)
 	}
 
 	for (i = 0; dev_list[i]; i++) {
-		if (strncmp(ibv_get_device_name(dev_list[i]), "mlx5", 4))
+		if (strncmp(ibv_get_device_name(dev_list[i]), "mlx5_0", 6))
 			continue;
 
 		if (!nic_pci_addr_str)
diff --git a/caladan/runtime/net/directpath/common.c b/caladan/runtime/net/directpath/common.c
index df11a39..1d315ef 100644
--- a/caladan/runtime/net/directpath/common.c
+++ b/caladan/runtime/net/directpath/common.c
@@ -52,7 +52,7 @@ REGISTER_CFG(directpath_pci_handler);
 size_t directpath_rx_buf_pool_sz(unsigned int nrqs)
 {
 	size_t buflen = MBUF_DEFAULT_LEN;
-	buflen *= MAX(8, guaranteedks) * (16 * RQ_NUM_DESC) * 2UL;
+	buflen *= MAX(24, guaranteedks) * (16 * RQ_NUM_DESC) * 2UL;
 	return align_up(buflen, PGSIZE_2MB);
 }
 
diff --git a/caladan/runtime/net/directpath/mlx5/mlx5_init.c b/caladan/runtime/net/directpath/mlx5/mlx5_init.c
index bf447ac..db8e0eb 100644
--- a/caladan/runtime/net/directpath/mlx5/mlx5_init.c
+++ b/caladan/runtime/net/directpath/mlx5/mlx5_init.c
@@ -379,7 +379,7 @@ int mlx5_common_init(struct hardware_q **rxq_out, struct direct_txq **txq_out,
 	}
 
 	for (i = 0; dev_list[i]; i++) {
-		if (strncmp(ibv_get_device_name(dev_list[i]), "mlx5", 4))
+		if (strncmp(ibv_get_device_name(dev_list[i]), "mlx5_0", 6))
 			continue;
 
 		if (!cfg_pci_addr_specified)
